vcf_annotation_tools/vcf_info_annotator.py

```diff
@@ -0,0 +1,96 @@
+import argparse
+import sys
+import vcfpy
+import csv
+from collections import OrderedDict
+
+def to_array(dictionary):
+    array = []
+    for key, value in dictionary.items():
+        array.append("{}|{}".format(key, value))
+    return sorted(array)
+
+def parse_tsv_file(args):
+    values={}
+    with open(args.values_file,'r') as tsvin:
+        tsvin = csv.reader(tsvin, delimiter='\t')
+        for row in tsvin:
+            values[(row[0] + ":" + row[1])] = row[2]
+    return values
+
+
+def create_vcf_reader(args):
+    vcf_reader = vcfpy.Reader.from_path(args.input_vcf)
+    if args.info_field in vcf_reader.header.info_ids():
+        vcf_reader.close()
+        raise Exception("INFO already contains a {} field. Choose a different label".format(args.info_field))
+    return vcf_reader
+
+def create_vcf_writer(args, vcf_reader):
+    new_header = vcf_reader.header.copy()
+    od = OrderedDict([('ID', args.info_field), ('Number', '.'), ('Type', args.value_format), ('Description', args.description)])
+    if args.source:
+        od['Source'] = args.source
+    if args.version:
+        od['Version'] = args.version
+    new_header.add_info_line(od)
+    output_file = args.output_vcf
+    return vcfpy.Writer.from_path(output_file, new_header)
+
+def define_parser():
+    parser = argparse.ArgumentParser("vcf-generic-annotator")
```

Comment:
What do you want the entry point to be names? Here it's `vcf-generic-annotator` but the file and class is VcfInfoAnnotator. This should be consistent.